R/diagnostics.R

```diff
@@ -144,7 +162,7 @@ print.pareto_k_table <- function(x, digits = 1, ...) {
 #' @return `pareto_k_ids()` returns an integer vector indicating which
 #' observations have Pareto \eqn{k} estimates above `threshold`.
 #'
-pareto_k_ids <- function(x, threshold = 0.5) {
+pareto_k_ids <- function(x, threshold = 0.7) {
```

Comment:
The documentation says the default `threshold` for `pareto_k_ids()` is `1 - 1 / log10(S)` but here the default seems to be `0.7`

Reply:
fixed